packages/formation/js/accordion.js

```diff
@@ -34,39 +34,44 @@ const loadAccordionHandler = () => {
     .forEach(element => {
       element.addEventListener('click', e => {
         const target = e.target;
-        const other = Array.from(
-          element.getElementsByClassName('usa-accordion-button'),
-        ).filter(item => item !== target);
 
-        const multiSelectable = toBoolean(
-          element.getAttribute('aria-multiselectable'),
-        );
+        // Checks whether the button has a click event already assigned to it.
+        // Specifically React Components.
+        if (!target.onclick) {
+          const other = Array.from(
```

Comment:
What is `other`?

Reply:
@rianfowler other is the other usa-accordion-buttons. Would otherButtons be better?

Reply:
are they nested or siblings? is it one or many?

Reply:
There are many. The current target is the <ul> accordion element and it will get the usa-accordion-button child elements. The child button element are inside of the <li> element but can be nested another level down. So they are not direct siblings.